README.rst

```diff
@@ -20,6 +20,8 @@ Plugin                               Status (*Experimental*, *Production*, or *D
 ===================================  ======================================================
 tutor-contrib-learner-dashboard-mfe  Experimental
 ===================================  ======================================================
+tutor-contrib-blockstore-minio       Experimental
```

Comment:
Missed this in the last one: the rows in the table shouldn't have separators between them.

```suggestion
tutor-contrib-blockstore-minio       Experimental
```

Reply:
working now https://github.com/openedx/openedx-tutor-plugins/blob/3c72af99c4a955eabe4e48e31710132dba5eb6a1/README.rst